### PR TITLE
blog: Add HA Node-RED

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Description

This blog post is aimed at:
1. Advertising the achiements in v1.4
2. Explain what would be requires to move beyond that
3. Discuss the trade-offs, and provide this blog to customers to understand what they need to think of

### CODE REVIEW

**Comment by ZJvandeWeg** [comment]
Draft as I want to add an image or two. Go beyond the wall of text

**Comment by ZJvandeWeg** [comment]
This PR now also includes some hacking to make the `image` tag work in blog posts. it renders like:

Screenshot 2023-02-08 at 12 16 30

----------

Index: 


Screenshot 2023-02-08 at 12 16 20

File: src/blog/2023/02/highly-available-node-red.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+as a fleet. Detecting failures is included in the install, and very fast. Comparing
+that to most alerting systems currently, it's usually a difference between night
+and day. Furthermore, to decrease the recovery time significantly from before
+there's a requirement to make software responsible for the whole procedure. Human
+need to just watch, they're much too slow.
+
+To get the MTTR down to 5 minutes there's a requirement to either make hardware
+automatically available to the fleet, or to over-provision (more hardware is
+available than there's an immediate need for). When a hardware failure occurs
+FlowForge is configured to ensure all Node-RED instances that are KIA are
```
[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/blog/2023/02/highly-available-node-red.md:55]: Killed in action - Very much needs a better word for it

**Comment by iskerrett** [comment]
FWIW, I wouldn't talk about this as a roadmap item in the title. I would use it to educate people on HA in Node-RED.

File: src/blog/2023/02/highly-available-node-red.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (iskerrett) [reply src/blog/2023/02/highly-available-node-red.md:55]: How about just say "Node-RED instance that fail are....

File: src/blog/2023/02/highly-available-node-red.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+that states 3 guarantees are wanted: Consistency, Availability, and Partition Tolerance. You get to pick
+only two. In manufacturing the line must never be stopped due to a software failure where possible,
+so Availability is one to pick. Now, do you want to drop Consistency? Meaning
+all 3 instances have the same view of global state? Or Partition Tolerance; when
+the network becomes unreliable (which it always is!) and partitioned all flows
+are moving onwards with the operations?
+
+### The roadmap
+
+With FlowForge v1.4, released February 2023, a 5 minute mean time to recovery is
```
[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/blog/2023/02/highly-available-node-red.md:84]: We don't manage the failover, but they could DIY it. BYO-load-balancer? Catchy :)

**Comment by robmarcer** [comment]
I feel I could make what I consider to be improvements to the english but I don't think they are vital, maybe it should sound like you've written it? Happy to approve, it all makes sense. If you want me to do a pass on the english let me know.

**Comment by ZJvandeWeg** [comment]
> I feel I could make what I consider to be improvements to the english but I don't think they are vital, maybe it should sound like you've written it? Happy to approve, it all makes sense. If you want me to do a pass on the english let me know.

@robmarcer please do a pass on the English so I can learn

File: src/blog/2023/02/highly-available-node-red.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/blog/2023/02/highly-available-node-red.md:84]: Not btw, hot-spares can already be setup using the FlowForge Device Agent. Not in Cloud instances

**Comment by robmarcer** [comment]
@ZJvandeWeg I'm done.

**Pull request merged**